packages/fe-fpm-writer/src/index.ts

```diff
@@ -8,7 +8,7 @@ export { CustomTableColumn } from './column/types';
 export { generateCustomColumn } from './column';
 
 export { CustomSection, CustomSubSection } from './section/types';
```

Comment:
I think additional exports are missing, like `CustomHeaderSection` and enums `RequestGroupId`, `DesignTime`